app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,8 @@
+class MenusController < ApplicationController
+
+  def show
+    @restaurant = Restaurant.find(params[:restaurant_id])
+    @menu = Menu.find_by(restaurant: @restaurant)
```

Comment:
This will return one (1) menu only. Are you sure about this? A restaurant can have many menus, right? If you are trying to find a specific menu you should look for it bu ID (passed in using params)